internal/presentation/presentation.go

```diff
@@ -75,7 +75,7 @@ func (o DepAnalysisOutput) Pretty(w io.Writer) error {
 		}
 	} else if len(o.Virtual) > 0 {
 		headers = []string{"Virtual Documents"}
-		rows = make([][]string, len(o.Base))
+		rows = make([][]string, len(o.Virtual))
```

Comment:
Are there any tests to add for this?

Reply:
I was hoping to get away with “no prior tests, so I don’t need to bother” 😄 Though I obviously tested this manually. 

I guess you’re right though, I can look into adding some later.